### PR TITLE
Add to research section to help clarify the issues with selects

### PR DESCRIPTION
**Pull request by CharlotteDowns**
Adding additional content to research section of the 'select' page to clarify the issues with selects [#2447](https://github.com/alphagov/govuk-design-system/issues/2447)

### CODE REVIEW

File: src/components/select/index.md.njk [hunk 1]
```diff
@@ -39,4 +39,16 @@ You can add hint text to help the user understand the options and choose one of
...
-Watch a video about how some [users struggle with selects](https://www.youtube.com/watch?v=CUkMCQR4TpY).
+User research has shown that some users struggle with selects.
+
+### Known issues and gaps
+
+Users have:
+  - been unable to close the select
+  - typed into the select
+  - confused focused items with selected items
+  - tried to pinch zoom select options on smaller devices
```
Comment (calvin-lau-sig7) [inline src/components/select/index.md.njk:50]: Might be better to give this list a bit more of an intro, how's this?

```suggestion
Research shows that users can struggle with selects, particularly when users have:
  - been unable to close the select
  - tried to type into the select
  - confused focused items with selected items
  - tried to pinch zoom select options on smaller devices
```

**Review by calvin-lau-sig7 — APPROVED** [review]

**Review by claireashworth — APPROVED** [review]

**Review by christopherthomasdesign — APPROVED** [review]
LGTM 👍

File: src/components/select/index.md [hunk 2]
```diff
@@ -39,4 +39,16 @@ You can add hint text to help the user understand the options and choose one of
 
 ## Research on this component
 
-Watch a video about how some [users struggle with selects](https://www.youtube.com/watch?v=CUkMCQR4TpY).
+User research has shown that some users struggle with selects.
+
+### Known issues and gaps
+
+Research shows that users can struggle with selects, particularly when users have:
+  - been unable to close the select
```
Comment (joelanman) [inline src/components/select/index.md:47]: one of the main issues I'm aware of is people not knowing that they can scroll the select or how to, worth adding?

Comment (CharlotteDowns) [reply src/components/select/index.md:47]: I'll add that in

**Pull request merged**